salt/modules/junos.py

```diff
@@ -1278,6 +1261,11 @@ def install_config(path=None, **kwargs):
             log.error(message)
             ret["message"] = message
             ret["out"] = False
+        except Exception as exc:  # pylint: disable=broad-except
+            ret["message"] = "install_config failed due to exception: '{}'".format(
+                exception
```

Comment:
```suggestion
                exc
```
The variable changed :)